test/e2e/live/native-runtime-qualification-case-executor.ts

```
@@ -0,0 +1,1373 @@
+        }),
+      );
+    }
+    try {
+      await stopService(service?.child ?? null, socket);
+    } catch (error) {
+      cleanupFailures.push(error);
+    }
+    service = null;
+    if (cleanupFailures.length > 0) {
+      if (qualificationFailure === undefined) {
+        throw new AggregateError(cleanupFailures, "Native runtime qualification cleanup failed");
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)